aries-site/src/pages/components/table.mdx

```diff
@@ -0,0 +1,243 @@
+quickly as possible.
+
+- Use Search. This is often the quickest, most efficient method for users to find a record of interest. 
+- As backup, use sortable columns coupled with paginated results. This combination 
+provides users with the knowledge of alpha-numeric range, enabling an 
+educated guess as to the record's location in the sort.
+
+<ExampleImagePreview
+  label="See finding a specific record"
+  href="#searching-and-filtering-tables"
+  src="/static/images/Searching_Data_Tables.png"
+  alt="Example demonstrating how to find a record of interest with a Table"
+/>
+
+### Narrowing a result set
+When a user is looking to focus on a group of records sharing similar characteristics, 
+providing thoughtful filter controls allows for incremental reduction of the table's 
+result set.
+
+- Provide filter controls.
+- Filter control input types should be chosen to best fit the data type and breadth of values.
+
+<ExampleImagePreview
+  label="Learn more about providing a great filter experience"
+  href="#searching-and-filtering-tables"
+  src="/static/images/Filtering_Data_Tables.png"
+  alt="Example demonstrating how to find a record of interest with a Table"
+/>
+
+### Browsing a result set
+Sometimes users are "checking up on things" or do not have specific records 
+in mind. In this case, they are assessing the state of their records and the 
+user experience objective is to facilitate the scanning of records to identify 
+potential outliers or records which the user is unaware need attention or action.
+
+- Use infinite scroll. In this scenario, infinite scroll removes friction, 
+allowing users to browse records effortlessly.
+- Consider adding column data types such as status indicators or meters 
+which call attention to records needing attention.
+
+<ExampleImagePreview
+  label="Using infinite scroll with tables"
+  href="#use-pagination-or-infinite-scroll-with-tables"
+  src="/static/images/Browsing_For_Outliers.png"
+  alt="Example demonstrating how to find a record of interest with a Table"
+/>
+
+### Assembling a data set
+Distinct from the user objectives above, tables containing data for analytical 
+purposes deserve distinct treatments as well. While tables are useful for 
+providing initial "discovery" type insights, such as identifying available attributes
+and ranges of values, purpose built tools are often a better choice for conducting 
+in depth analysis. In this scenario, a table's design should be optimized for 
+assembling the data set for analysis and easy portability of that data into the 
+user's preferred analytical tool.
+
+- Provide [filter controls](#searching-and-filtering-tables).
+- Filter control input types should be chosen to best fit the data type and breadth of values.
+- Provide export capability for users to consume the data in their analytical 
+tool of choice.
+- Provide aggregate measures in the [table footer](#column-summaries-and-aggregation).
+
+<!--
+## Guidelines
```

Comment:
Noticing you left this commented out, is this because you want more team input here? or would we want to include it

Reply:
I think it is an area on which we should have perspective, however, I think we haven't done enough of the background work here. I left this here to reference and potentially bring in in the future.

Reply:
Sounds good to me.